google/cloud/bigtable/tests/instance_admin_emulator.cc

```diff
@@ -42,6 +42,13 @@ class InstanceAdminEmulator final
     google::protobuf::TextFormat::PrintToString(*request, &request_text);
     std::cout << __func__ << "() request=" << request_text << "\n";
 
+    constexpr int kMaxInstanceIdLength = 33;
+    constexpr int kMinInstanceIdLength = 6;
+    if (((request->instance_id()).size() > kMaxInstanceIdLength) &&
+        ((request->instance_id()).size() < kMinInstanceIdLength)) {
+      throw std::invalid_argument(
```

Comment:
Raw `throw`s don't compile in the "noex" builds.  Consider `google/cloud/internal/throw_delegate.h`.